lib/openapi_rspec/module_helpers.rb

```diff
@@ -15,6 +15,10 @@ def params(&block)
       let(:openapi_rspec_params, &block)
     end
 
+    def body(&block)
```

Comment:
`raw_body` should describe it better, WDYT?

Reply:
sure. why not. 
The other option would be to mimic rspec_rails (?) and allow String input for `params`. It probably makes the code a bit more complicate but reduced the confusion about "raw body"  and "params (body)" input. 
What's your tale on that?

Reply:
It's even better =)

Reply:
Done :)